addons/mail/static/src/components/discuss/tests/discuss_inbox_tests.js

```diff
@@ -550,6 +552,35 @@ QUnit.test('click on (non-channel/non-partner) origin thread link should redirec
     assert.verifySteps(['do-action'], "should have made an action on click on origin thread (to open form view)");
 });
 
+QUnit.test('subject should not be shown when subject is same as thread name', async function (assert) {
```

Comment:
```suggestion
QUnit.test('subject should not be shown when subject differs from thread name only by the "Re:" prefix', async function (assert) {
```

Reply:
You can also make another test for the basic case, without prefix at all. That is named like the current test.